packages/react/src/components/header/Header.stories.tsx

```diff
@@ -709,3 +712,170 @@ export const WithFullFeaturesCustomTheme = (args) => {
+            />
+          </Header.ActionBarItem>
+          <Header.ActionBarItem icon={<IconCogwheel />} ariaLabel="Test items">
+            <Header.ActionBarSubItem label="Link" href="/" />
+            <Header.ActionBarSubItem label="Link" href="/" />
+            <Header.ActionBarSubItem label="Link" href="/" />
+          </Header.ActionBarItem>
+          <Header.ActionBarItem label="Another Test">
+            <Header.ActionBarSubItem label="Link" href="/" />
+            <Header.ActionBarSubItem label="Link" href="/" iconLeft={<IconCogwheel />} />
+            <Header.ActionBarSubItem
+              label="Link"
+              href="/"
+              iconLeft={<IconCogwheel />}
+              iconRight={<IconUser />}
+              external
+            />
+            <Header.ActionBarSubItem
+              label="Link"
+              href="/"
+              iconLeft={<IconCogwheel />}
+              external
+              notificationCountAriaLabel="Uusia viestejä"
+              notificationCount={2}
+            />
+            <Header.ActionBarSubItemGroup label="Heading">
+              <Header.ActionBarSubItem
+                label="Link"
+                href="/"
+                notificationCountAriaLabel="Uusia käyttäjiä"
+                notificationCount="3"
+              />
+              <Header.ActionBarSubItem label="Link" href="/" bold />
+              <Header.ActionBarSubItem
+                label="Link"
+                href="/"
+                notificationCountAriaLabel="Tehtäviä tekemättä"
+                notificationCount={822}
+              />
+            </Header.ActionBarSubItemGroup>
+            <Header.ActionBarSubItemGroup label="Heading" iconLeft={<IconCogwheel />}>
+              <Header.ActionBarSubItem label="Link" href="/" />
+            </Header.ActionBarSubItemGroup>
+            <Header.ActionBarSubItemGroup label="Subheading" href="/" iconLeft={<IconCogwheel />}>
+              <Header.ActionBarSubItem label="Link" href="/" />
+              <Header.ActionBarSubItem label="Link" href="/" />
+            </Header.ActionBarSubItemGroup>
+          </Header.ActionBarItem>
+          <Header.ActionBarItem icon={<IconCogwheel />} label="Test">
+            <Header.ActionBarSubItem label="Static" />
+            <Header.ActionBarSubItem label="Link" href="/" />
+            <Header.ActionBarSubItem label="External Link" href="/" external />
+            <Header.ActionBarSubItem label="Link" href="/" iconLeft={<IconCogwheel />} />
+            <Header.ActionBarSubItem label="Link" href="/" iconLeft={<IconUser />} iconRight={<IconCogwheel />} />
+            <Header.ActionBarSubItem
+              label="External Link with right icon and with too long description text"
+              href="/"
+              external
+              iconLeft={<IconUser />}
+              iconRight={<IconCogwheel />}
+              notificationCountAriaLabel="Tehtäviä tekemättä"
+              notificationCount="2"
```

Comment:
I get TS-error "notificationCount should be a number".

"string" might actually be better if someone wants to add for example "!"?

Reply:
But then it cannot be "notificationCount" any more 😬

Reply:
Yeah, it would be some `notificationBubbleContent` etc :D

Reply:
This still needs documentation